openedx/core/djangoapps/enrollments/api.py

```diff
@@ -527,6 +535,23 @@ def is_enrollment_valid_for_proctoring(username, course_id):
     if enrollment['mode'] not in appropriate_modes:
         return False
 
+    # Only continue if a proctored exam is found in the course
+    has_proctored_exam = False
+
+    timed_exams = modulestore().get_items(
+        course_id,
+        qualifiers={'category': 'sequential'},
+        settings={'is_time_limited': True}
+    )
+
+    for exam in timed_exams:
+        if exam.is_proctored_exam:
+            has_proctored_exam = True
+            break
+
+    if not has_proctored_exam:
```

Comment:
Can this be simplified to just `return has_proctored_exam`?

Reply:
Good call!